`titleside` is removed from plotly version 6

Error after updating the Python lib plotly:

```bash
INFO:root:Building the dashboard
INFO:root:RUNNING: make dashboard ROBOT_JAR=/tools/robot.jar ROBOT=robot  -B (Timeout: 3600)
INFO:root:OUT: b'mkdir -p dashboard/assets
curl -Lk -o dashboard/assets/check.svg https://raw.githubusercontent.com/iconic/open-iconic/master/svg/check.svg
curl -Lk -o dashboard/assets/info.svg https://raw.githubusercontent.com/iconic/open-iconic/master/svg/info.svg
curl -Lk -o dashboard/assets/warning.svg https://raw.githubusercontent.com/iconic/open-iconic/master/svg/warning.svg
curl -Lk -o dashboard/assets/x.svg https://raw.githubusercontent.com/iconic/open-iconic/master/svg/x.svg
python3 util/create_dashboard_html.py dashboard dependencies/ontologies.yml dashboard-config.yml ""dashboard/dashboard-results.yml"" "ROBOT version 1.9.8" ""https://github.com/OBOFoundry/OBO-Dashboard/commit/fae181949e56485fd44c77ae98050c4fa993dd6f"" dashboard/index.html
python3 util/md_to_html.py docs/about.md -t util/templates/about.html.jinja2 -o dashboard/about.html
python3 util/dashboard_analysis_html.py --dashboard-results "dashboard/dashboard-results.yml" --template util/templates/analysis.html.jinja2 --output dashboard/analysis.html
'
INFO:root:ERROR: b'Makefile:168: warning: overriding recipe for target \'build/robot.jar\'
Makefile:44: warning: ignoring old recipe for target \'build/robot.jar\'
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100   233  100   233    0     0  10[27](https://github.com/OBOFoundry/dashboard-template/actions/runs/15371681230/job/43251626076#step:7:28)5      0 --:--:-- --:--:-- --:--:-- 10590
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100   367  100   367    0     0  16181      0 --:--:-- --:--:-- --:--:-- 16681
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100   312  100   312    0     0  13949      0 --:--:-- --:--:-- --:--:-- 14181
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100   [28](https://github.com/OBOFoundry/dashboard-template/actions/runs/15371681230/job/43251626076#step:7:29)7  100   287    0     0  12412      0 --:--:-- --:--:-- --:--:-- 12478
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0\r100  135k    0  135k    0     0   753k      0 --:--:-- --:--:-- --:--:--  750k
Traceback (most recent call last):
  File "/tools/OBO-Dashboard/util/dashboard_analysis_html.py", line 405, in <module>
    main(sys.argv)
  File "/tools/OBO-Dashboard/util/dashboard_analysis_html.py", line 396, in main
    plot_graph_dependency(*graph_to_plot_dependency(dash_results))
  File "/tools/OBO-Dashboard/util/dashboard_analysis_html.py", line 287, in plot_graph_dependency
    node_trace = go.Scatter(
                 ^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/plotly/graph_objs/_scatter.py", line 2705, in __init__
    self._set_property("marker", arg, marker)
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 4422, in _set_property
    _set_property_provided_value(self, name, arg, provided)
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 398, in _set_property_provided_value
    obj[name] = val
    ~~~^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 4944, in __setitem__
    self._set_compound_prop(prop, value)
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 5355, in _set_compound_prop
    val = validator.validate_coerce(val, skip_invalid=self._skip_invalid)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/_plotly_utils/basevalidators.py", line 2489, in validate_coerce
    v = self.data_class(v, skip_invalid=skip_invalid, _validate=_validate)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/plotly/graph_objs/scatter/_marker.py", line 1113, in __init__
    self._set_property("colorbar", arg, colorbar)
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 4422, in _set_property
    _set_property_provided_value(self, name, arg, provided)
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 398, in _set_property_provided_value
    obj[name] = val
    ~~~^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 4944, in __setitem__
    self._set_compound_prop(prop, value)
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 5355, in _set_compound_prop
    val = validator.validate_coerce(val, skip_invalid=self._skip_invalid)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/_plotly_utils/basevalidators.py", line 2489, in validate_coerce
    v = self.data_class(v, skip_invalid=skip_invalid, _validate=_validate)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/plotly/graph_objs/scatter/marker/_colorbar.py", line 1724, in __init__
    self._process_kwargs(**dict(arg, **kwargs))
  File "/usr/local/lib/python3.12/dist-packages/plotly/basedatatypes.py", line 4470, in _process_kwargs
    raise err
ValueError: Invalid property specified for object of type plotly.graph_objs.scatter.marker.ColorBar: \'titleside\'

Did you mean "title"?

Bad property path:
titleside
^^^^^^^^^
make: *** [Makefile:164: dashboard/analysis.html] Error 1
'
Traceback (most recent call last):
  File "/tools/OBO-Dashboard/./util/dashboard_config.py", line 481, in <module>
    cli()
  File "/usr/local/lib/python3.12/dist-packages/click/core.py", line 1442, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/click/core.py", line 1363, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/click/core.py", line 18[30](https://github.com/OBOFoundry/dashboard-template/actions/runs/15371681230/job/43251626076#step:7:31), in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/click/core.py", line 1226, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/dist-packages/click/core.py", line 794, in invoke
    return callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/tools/OBO-Dashboard/./util/dashboard_config.py", line 77, in rundashboard
    runcmd(f"make dashboard {make_parameters} -B", config.get_dashboard_report_timeout_seconds())
  File "/tools/OBO-Dashboard/util/lib.py", line 47, in runcmd
    command.run(timeout=timeout)
  File "/tools/OBO-Dashboard/util/lib.py", line [42](https://github.com/OBOFoundry/dashboard-template/actions/runs/15371681230/job/43251626076#step:7:43), in run
    raise Exception(f'Failed: {self.cmd} with return code {self.process.returncode}')
Exception: Failed: make dashboard ROBOT_JAR=/tools/robot.jar ROBOT=robot  -B with return code 2
Error: Process completed with exit code 1.
```

---

`titleside` is now part of the `title` object in colorbar object.

Reference: https://plotly.com/python/reference/scatter/#scatter-marker-colorbar-title

--- a/util/dashboard_analysis_html.py
+++ b/util/dashboard_analysis_html.py
@@ -300,9 +300,11 @@ def plot_graph_dependency(
             "size": node_sizes,
             "colorbar": {
                 "thickness": 15,
-                "title": "Node Connections",
+                "title": {
+                    "text": "Node Connections",
+                    "side": "right",
+                },
                 "xanchor": "left",
-                "titleside": "right"
             },
             "line_width": 2
         }

